Makefile

```
@@ -13,11 +13,21 @@ watch:
 build-prod:
 	mkdir -p prod
 	cp dev/index.html prod
-	rm -rf dist
+	twpurs gen-css --src src/Page --css prod/tailwind.css
```

With the new version of tailwind-purs we can remove this
```suggestion
	twpurs gen-css --css prod/tailwind.css
```